design/20240122.scarf.md

```diff
@@ -0,0 +1,62 @@
+# Proposal: Scarf.sh integration
+
+<!-- toc -->
+- [Summary](#summary)
+  - [What is Scarf?](#what-is-scarf)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+- [Proposal](#proposal)
+  - [How will this change impact our users?](#how-will-this-change-impact-our-users)
+  - [Known issues/limitations](#known-issueslimitations)
+<!-- /toc -->
+
+## Summary
+
+With our focus on CNCF graduation, CNCF aims for its projects to become [vendor-neutral](https://contribute.cncf.io/maintainers/community/vendor-neutrality/) wherever possible. The cert-manager project should uphold this aim. In doing so, it will need to take a further step to move on from its proud Jetstack legacy with a change to remove Jetstack from the container image repository name.
+
+In addition, Quay.io, the current container image registry for cert-manager, has limitations on the amount of analytic data it can provide due to the high volume of downloads that cert-manager receives. The cert-manager maintainers have also found that Quay has had several outages during 2023, and they want to manage that situation quickly in the future if required.
```

Comment:
scarf.sh also seems to have had several outages during 2023
![image](https://github.com/cert-manager/cert-manager/assets/978965/d027ff59-3809-4693-a50e-d24e0a625fd0)

But there's no record of these on their uptime history page AFAICS:
 * https://status.scarf.sh/history

Reply:
Hi @wallrj , just chiming in here, the Scarf team is working to backfill those incidents and they should be up shortly. We will comment again to update when finished.

Reply:
Thanks Melanie. I should also have linked to this issue:
 * https://github.com/scarf-sh/gateway/issues/26

Reply:
@wallrj https://github.com/scarf-sh/gateway/issues/26 has been updated. We've officially switched over our status page provider and backfilled the incident reports for the Gateway.